ci/travis/lib.sh

```diff
@@ -5,6 +5,38 @@ NUM_JOBS=4
 echo_red() { printf "\033[1;31m$*\033[m\n"; }
 echo_green() { printf "\033[1;32m$*\033[m\n"; }
 
+############################################################################
+# Check if the file is not present in .clangformatignore in order wheter or 
+# not the file is formatted properly
+############################################################################
+is_valid_file() {
```

Comment:
The name 'is_valid_file' does not reflect the implementation. A more descriptive name can be 'not_in_clang_ignore_list' but I'm open to other suggestions.